app/src/main/java/com/madonasyombua/sportsdb/data/remote/response/LeagueTableResponse.kt

```diff
@@ -0,0 +1,9 @@
+package com.madonasyombua.sportsdb.data.remote.response
+
+import com.madonasyombua.sportsdb.data.remote.model.LeagueTable
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class LeagueTableResponse(
+    val table:List<LeagueTable>
+)
```

Comment:
Do we need to `serialize` this?

Reply:
not necessary only integer is been passed around